woodwork/utils.py

```diff
@@ -298,3 +299,149 @@ def _parse_logical_type(logical_type, name):
         return logical_type
     else:
         raise TypeError(f"Invalid logical type specified for '{name}'")
+
+
+def concat(objs, axis=1, join='outer', validate_schema=True):
+    """
+    Concatenate Woodwork objects along a particular axis. Cannot concatenate DataFrames
+    with conflicting Woodwork index or time index columns. If DataFrames have the same Woodwork
+    index or time index columns, the first appearance of the column will be included in the
+    concatenated DataFrame. As Woodwork does not allow duplicate column names,
+    will not allow duplicate columns at concatenation.
```

Comment:
I'm not sure how exactly best to do this, but I'm thinking that some of the behaviors between our concat and pandas concat are different (for example how we deal with a set ww index that is duplicated). It might be good for us to show some typical use cases to show how this concat works, similar to how pandas has several examples in their [documentation](https://pandas.pydata.org/docs/reference/api/pandas.concat.html). 

Maybe for now we could add some examples to our documentation page for this function, like pandas has done, unless you have another idea.

Reply:
with the changes, the main difference is how we restrict to just allowing concatenation of columns and that we don't allow duplicate columns. 

I think these might be a little harder to showcase in examples, so it might be better suited for a guide or somewhere where we have more space to explain. Not totally sure though

Reply:
I'm thinking examples probably aren't as needed with this updated version since there are fewer ways this function can be used now.